docs/CONTRIBUTING.rst

```diff
@@ -262,6 +262,38 @@ Following [semantic versioning](https://semver.org/), increment the:
 * ``MINOR`` version when you add functionality in a backwards compatible manner, and
 * ``PATCH`` version when you make backwards compatible bug fixes.
 
+Interpreting version numbers
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The version number is made available to users as an attribute of the module:
+
+.. code:: python
+
+   >>> import resqpy
+   >>> print(resqpy.__version__)
+   '1.6.1'
+
+When working with a development version of the code that does not correspond to
+a tagged release, the version number will look a little different, for example
+``1.6.2.dev301+gddfbf6c``.
+
+This can be interpreted as:
+
+* ``1.6.2`` : is the *next* expected release. The previous release would be ``1.6.1``.
+* ``dev301`` : 301 commits added since the previous release.
+* ``+gddfbf6c`` : a ``+g`` prefix followed by current commit ID: ``dfbf6c``.
```

Comment:
Should commit ID not be 'ddfbf6c'? Looks all good apart from that

Reply:
Thanks - fixed.